Makefile

```diff
@@ -15,6 +15,8 @@ SUBDIRS:=agent runtime snapshotter
 export INSTALLROOT?=/usr/local
 export STATIC_AGENT
 
+GOPATH:=$(shell go env GOPATH)
```

Comment:
This should not be needed.

Reply:
:+1: I'd prefer to install this into a local `.gitignore`d directory instead of the user's `GOPATH`.

Reply:
Never mind; I understand what's going on. We still need GOPATH to expand to `~/go` even if $GOPATH isn't explicitly set. I misunderstood your intent.

Reply:
We already install `git-validation` and `ltag` to user's `GOPATH`.
I don't think moving `golangci-lint` would help a lot.

Reply:
:frowning_face: I'd prefer not to install any of those into the user's `GOPATH`.

Reply:
Let's solve this separately. The intent of this PR was to fix GOPATH in one specific case.

Reply:
That's fine; we can table the discussion for now and move on.  But I'm going to continue to be unhappy about a Makefile target that mutates the contents of my GOPATH.